src/base/bittorrent/session.cpp

```diff
@@ -109,6 +114,10 @@ namespace
     void torrentQueuePositionTop(const libt::torrent_handle &handle);
     void torrentQueuePositionBottom(const libt::torrent_handle &handle);
 
+#ifdef Q_OS_WIN
+    QString convertIfaceNameToGuid(const QString &name);
```

Comment:
I would put this function in anonymous namespace (or in Utils::Net if it can be reused).